litmus/director/openebs-upgrade-jiva/busybox_app.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: busybox
+  labels:
+    app: bb
+spec:
+  selector:
+    matchLabels:
+      app: bb
+  template:
+    metadata:
+      labels:
+        app: bb
+    spec:
+      containers:
+      - name: app-busybox
+        imagePullPolicy: IfNotPresent
+        image: busybox
```

Comment:
please check if busybox can be considered as a valid application?
cc @ajeshbaby

Reply:
ok

Reply:
Changed to mongo statefulset